src/Entity/EntityBase.php

```diff
@@ -147,10 +147,7 @@ public static function baseFieldDefinitions(EntityTypeInterface $entity_type) {
       ->setDescription(t('The entity type.'))
       ->setSetting('target_type', $entity_type->getBundleEntityType());
 
-    $fields['revision_log'] = BaseFieldDefinition::create('string_long')
-      ->setLabel(t('Revision log message'))
-      ->setDescription(t('The log entry explaining the changes in this revision.'))
-      ->setRevisionable(TRUE);
+    $fields += static::entityRevisionLogBaseFieldDefinitions();
```

Comment:
entityRevisionLogBaseFieldDefinitions adds base fields that are look to be redundant. The entity_revision table already has revision_timestamp, revision_uid and revision_log. It also looks like these fields have been removed in the 8.x-2.x verison of the entity module. This pull request also results in a duplicate log message field. Is there a reason we need these fields added to the entity_data table?